packages/web/components/bridge/use-bridge-quotes.ts

```diff
@@ -422,23 +424,42 @@ export const useBridgeQuotes = ({
 
   const [transferInitiated, setTransferInitiated] = useState(false);
   const trackTransferStatus = useCallback(
-    (providerId: Bridge, params: GetTransferStatusParams) => {
+    ({
+      estimatedArrivalUnix,
+      providerId,
+      params,
+    }: {
+      estimatedArrivalUnix: number;
+      providerId: Bridge;
+      params: GetTransferStatusParams;
+    }) => {
       if (inputAmountRaw !== "" && availableBalance && inputCoin) {
-        transferHistoryStore.pushTxNow(
-          `${providerId}${JSON.stringify(params)}`,
-          inputCoin.trim(true).toString(),
+        transferHistoryStore.pushTxNow({
+          prefixedKey: `${providerId}${JSON.stringify(params)}`,
+          amount: inputCoin.trim(true).toString(),
+          amountLogo: isWithdraw ? toAsset?.imageUrl : fromAsset.imageUrl,
           isWithdraw,
-          (isWithdraw ? fromAddress : toAddress) ?? "" // use osmosis account (destinationAddress) for account keys (vs any EVM account)
-        );
+          chainPrettyName:
+            direction === "deposit"
+              ? fromChain?.chainName ?? ""
```

Comment:
I think if these are the display chain type, we should use the `prettyName` instead so we can have capitalization